plugins/uniques.js

```diff
@@ -167,9 +218,13 @@ window.plugin.uniques.updateCheckedAndHighlight = function(guid) {
 
 		var uniqueInfo = plugin.uniques.uniques[guid],
 			visited = (uniqueInfo && uniqueInfo.visited) || false,
-			captured = (uniqueInfo && uniqueInfo.captured) || false;
+			captured = (uniqueInfo && uniqueInfo.captured) || false,
+			scouted = (uniqueInfo && uniqueInfo.scouted) || false,
+			droned = (uniqueInfo && uniqueInfo.droned) || false;
```

Comment:
Still unresolved. Properties, ids, ...

Reply:
code and properties changed:
- scouted -> scoutControlled
- droned ->droneVisited

converter implemented. It will add a version info to the uniques MAP.